Playground/RxSwiftExtPlayground.playground/Pages/zipWith.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,37 @@
+/*:
+ > # IMPORTANT: To use `RxSwiftExtPlayground.playground`, please:
+
+1. Make sure you have [Carthage](https://github.com/Carthage/Carthage) installed
+1. Fetch Carthage dependencies from shell: `carthage bootstrap --platform ios`
+1. Build scheme `RxSwiftExtPlayground` scheme for a simulator target
+1. Choose `View > Show Debug Area`
+ */
+
+//: [Previous](@previous)
+
+import RxSwift
+import RxSwiftExt
+
+/*:
+ ## zip(with:)
+
+ Merges the specified observable sequences into one observable sequence by using the selector function whenever all
+ of the observable sequences have produced an element at a corresponding index.
+
+ */
+example("zip values") {
+
+    let numbers = Array<Int>([1, 2, 3])
+    let strings = Array<String>(["a", "b", "c"])
+    Observable.from(numbers)
+            .zip(with: Observable.from(strings)) { i, s in
+                s + String(i)
+            }
+            .toArray()
+            .subscribe(onNext: { result in
+                // look types on the right panel ===>
```

Comment:
Not sure what this comment means, but I think it needs rephrasing :)

Reply:
updated